x/wasm-storage/keeper/msg_server.go

```diff
@@ -65,32 +65,29 @@ var _ types.MsgServer = msgServer{}
 // checks if a module with the same hash already exists, sets the Wasm instance in the data store,
 // tracks its expiration, emits an event, and returns the hash of the stored Wasm module.
 func (m msgServer) StoreDataRequestWasm(goCtx context.Context, msg *types.MsgStoreDataRequestWasm) (*types.MsgStoreDataRequestWasmResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
```

Comment:
unnecessary unwrap. If `ValidateBasic` fails.

Reply:
well noted - will address in another PR as it's not related to this PR